emission/net/api/utility_model_api.py

```diff
@@ -0,0 +1,54 @@
+import emission.user_model_josh.utility_model as eum
+import emission.core.get_database as edb
+
+import json
+
+def write_to_db_from_browser(info):
+    db = edb.get_utility_model_db()
+    at = int((datetime.datetime.utcnow() - datetime.datetime.utcfromtimestamp(0)).total_seconds())
+    value_line = value_line.split(";")
+    start = value_line[0]
+    end = value_line[1]
+
+    start = base.geocode_with_cache(start)
+    end = base.geocode_with_cache(end)
+    
+    time_info = {}
+    if value_line[2] == "leaveNow":
+        time_info["leave"] = True
+        time_info["when"] = datetime.datetime.now()
+        print "leaveNow"
+    elif value_line[2] == "leaveAt":
+        time_info["leave"] = True
+        time_info["when"] = str_time_to_datetme(value_line[3])
+        print "leaveAt"
+    elif value_line[2] == "thereBy":
+        time_info["leave"] = False
+        time_info["when"] = str_time_to_datetme(value_line[3])
+        print "arriveAt"
+
+    bike = get_bike_info(value_line[4])
+
+    user = UserModel("scottMoura", bike)
```

Comment:
Why is this hardcoded? Won't your user counting be broken if you always use the same user?